railib/api.py

```diff
@@ -323,6 +324,43 @@ def run(self, ctx: Context, *args) -> dict:
         rsp = rest.post(ctx, url, data, **kwargs)
         return json.loads(rsp)
 
+#
+# /transactions endpoint
+#
+class TransactionAsync(object):
```

Comment:
im surprised this required another Transactions object .. this object is really just a helper to construct the complex v1 payload, which we are re-using for the first phase of the v1 transaction endpoint? is it because it embeds the /transaction path?

Reply:
Resolving this based on the discussion on Dec 10.